cumulusci.yml

```diff
@@ -99,21 +99,21 @@ tasks:
 
     robot:
         options:
-            suites: robot/HEDAP/tests
+            suites: robot/EDAP/tests
             options:
-                outputdir: robot/HEDAP/results
+                outputdir: robot/EDAP/results
 
     robot_libdoc:
         options:
-            path: robot/HEDAP/resources/HEDA.py,robot/HEDAP/resources/HEDA.robot
-            output: robot/HEDAP/doc/Keywords.html
+            path: robot/EDAP/resources/EDA.py,robot/EDAP/resources/EDA.robot
+            output: robot/EDAP/doc/Keywords.html
 
     update_admin_profile:
         class_path: tasks.salesforce.UpdateAdminProfile
 
 flows:
-    hedaectomy:
```

Comment:
Is this going to break anything? Or is it okay to rename?

Reply:
I restored the 'h'